caikit/runtime/servicers/global_predict_servicer.py

```diff
@@ -158,61 +158,58 @@ def Predict(
         model_id = get_metadata(context, self.MODEL_MESH_MODEL_ID_KEY)
         request_name = caikit_rpc.request.name
 
-        with self._handle_predict_exceptions(model_id, request_name):
-            with alog.ContextLog(
-                log.debug, "GlobalPredictServicer.Predict:%s", request_name
-            ):
-                # Retrieve the model from the model manager
-                log.debug("<RUN52259029D>", "Retrieving model '%s'", model_id)
-                model = self._model_manager.retrieve_model(model_id)
-                model_class = type(model)
+        with self._handle_predict_exceptions(model_id, request_name), alog.ContextLog(
```

Comment:
Magic! I did not realize you could chain `with` statements! This changes everything about gorpy extra indentation